commands/doctor.ts

```diff
@@ -0,0 +1,73 @@
+import { trackCommandUsage } from '../lib/usageTracking';
+import { logger } from '@hubspot/local-dev-lib/logger';
+import fs from 'fs';
+import { Doctor } from '../lib/doctor/Doctor';
+import { EXIT_CODES } from '../lib/enums/exitCodes';
+import path from 'path';
+import { ArgumentsCamelCase, BuilderCallback, Options } from 'yargs';
+import { getCwd } from '@hubspot/local-dev-lib/path';
+const { i18n } = require('../lib/lang');
+
+export interface DoctorOptions {
+  'output-dir'?: string;
+}
+
+const i18nKey = 'commands.doctor';
+
+export const command = 'doctor';
+export const describe = i18n(`${i18nKey}.describe`);
+
+export const handler = async ({
+  outputDir,
+}: ArgumentsCamelCase<DoctorOptions>) => {
+  const doctor = new Doctor();
+
+  try {
+    trackCommandUsage('doctor', undefined, doctor.accountId);
```

Comment:
The trackCommandUsage command itself should handle the try/catch and debugging. So you should be able to just call it directly without the wrapper.